test/fixture/validate-installed-global.js

```diff
@@ -0,0 +1,3 @@
+import test from '../../'
```

Comment:
@jamestalmage Yeah, just XO that ignores it because it's fixtures. https://github.com/sindresorhus/xo/blob/0a5060c717876f77d9bcc9b8b453ba14eec332f1/index.js#L20 Starting to question that.